### PR TITLE
Improve highlighting of table focused/hovered/selected row 

### PR DESCRIPTION
**Pull request by VkoHov**
- Change row color styles
- Add functionality to separate hovered and focused rows
- Change rows group header styles when it's grouped by color

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#1261](https://github.com/aimhubio/aim/issues/1261)

File: aim/web/ui/src/pages/ImagesExplore/ImagesExplore.tsx [hunk 1]
```diff
@@ -365,6 +365,10 @@ function ImagesExplore(): React.FunctionComponentElement<React.ReactNode> {
                       ? 'medium'
                       : 'large'
                   }
+                  focusedState={
+                    imagesExploreData?.config?.images
+                      ?.focusedState as IFocusedState
+                  }
```
Comment (arsengit) [inline aim/web/ui/src/pages/ImagesExplore/ImagesExplore.tsx:371]: You can change type here with  ! operator.

**Review by arsengit — CHANGES_REQUESTED** [review]
There is a bug when you switching 8 to 24 color palette color set from grouping by color popover, the column background becoming transparent.

File: aim/web/ui/src/config/table/tableConfigs.ts [hunk 2]
```diff
@@ -19,4 +19,7 @@ export const rowCeilSizeConfig = {
   },
 };
 
+export const tableColoredSelectionColumnWidth = 50;
```
Comment (arsengit) [inline aim/web/ui/src/config/table/tableConfigs.ts:22]: @VkoHov what do you think about changing name to uppercase and separate with underlines. 
For instance COLORED_SELECTION_COLUMN_WIDTH.

**Review by arsengit — APPROVED** [review]

**Review by Hamik25 — CHANGES_REQUESTED** [review]

**Review by roubkar — APPROVED** [review]

**Review by Hamik25 — APPROVED** [review]

**Pull request merged**